src/js/console.js

```diff
@@ -18,7 +18,23 @@ var util = require('util');
 
 
 function Console() {
-  // Empty
+  var prop = {
+    writable: true,
+    enumerable: false,
+    configurable: true
+  };
+  prop.value = stdout;
```

Comment:
Don't use the same property object for different defines, use `Object.assign()` to generate separate property :)